masuite-common/build.gradle.kts

```diff
@@ -31,15 +31,19 @@ java {
     targetCompatibility = JavaVersion.toVersion(16)
 }
 
-tasks.create<ConfigureShadowRelocation>("relocateShadowJar") {
-    target = tasks["shadowJar"] as ShadowJar
-    prefix = "dev.masa.masuite.libs"
+tasks.withType(AbstractArchiveTask::class).configureEach {
+    isReproducibleFileOrder = true
+    isPreserveFileTimestamps = false
 }
 
-tasks.withType<ShadowJar> {
-    dependsOn(tasks["relocateShadowJar"])
+tasks.withType<com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar> {
+    val prefix = "dev.masa.masuite.libs.gson."
 
-    mergeServiceFiles()
+    relocate("com.google.gson", prefix + "gson")
```

Comment:
Do we need to relocate this? I think it was just an example from me?

Reply:
Shouldn't be everything relocated that is build into the jar?

Reply:
Velocity, Waterfall / BungeeCord, Paper (and Sponge) already incude GSON. It seems like I've accidentally added 
`implementation("com.google.code.gson:gson:2.8.9")` instead of `compileOnly("com.google.code.gson:gson:2.8.9")`.

Reply:
will change it